lib/bundlex/cnode.ex

```diff
@@ -88,11 +95,22 @@ defmodule Bundlex.CNode do
     do_start(app, native_name, false)
   end
 
+  defp cookie_env?(app,native_name) do 
+    cookie_env = Application.get_env(app, :cookie_env)
+    cond do 
+      is_boolean(cookie_env) -> cookie_env
+      is_map(cookie_env) && is_boolean(cookie_env[native_name]) -> cookie_env[native_name]   
+      is_nil(cookie_env) && app == :bundlex -> false
+      app != :bundlex -> cookie_env?(:bundlex,native_name)
```

Comment:
The code to resolve the config option is a bit complex,
i chose it to allow partial migrations in code using this.

the code above resolves the `cookie_env` option starting at the `app` config.
and if it's a map it loops up the native_name.
If it's not set, it gives the `:bundlex` config a try.

If per executable config is overkill, this can be dropped my (first commit should have this version).

I personally would never pass the cookie per argument, so i'll be setting `config :bundlex :cookie_env true` for all of my projects.
but this is a breaking change so it's not what i would suggest. 

---

Alternatively i would suggest to enforce an global or per-app config, and __fail__ if no option is set (to force users to choose an option).

It reduces complexity and improves security (by notifying explicitly about this option).
But it would force users to set the configuration upon update.

After thinking a bit about this...
For our commercial projects, and maybe even for public libraries I would maintain, I would probably chose a less configurable option and enforce a configuration.

Reply:
Oh, now I see. Acctually bundlex hasn't been released as stable yet. It's still in initial development phase so making breaking changes shouldn't cost us much. For me switching entirely to passing cookie via env variable sounds even better but let's wait for @mat-hek with decision. He will be back on Monday so I think we can merge it then.